ChartboostMediationDemo/Demo_SwiftUI/AdTypeSelectionView.swift

```diff
@@ -11,11 +11,13 @@
 //
 
 import SwiftUI
+import ChartboostMediationSDK
 
 /// A view that lists the different Chartboost Mediation SDK advertisement types. Selecting one will
 /// navigate to a view that can be used to load and show that type of advertisement.
 struct AdTypeSelectionView: View {
     @State private var useFullscreenApi = true
+    private let adQueueViewModel: AdQueueViewModel = AdQueueViewModel()
```

Comment:
This being a regular property in a View means it will be recreated several times when the SwiftUI system checks if view updates are needed, which is probably not what we want.
The way to solve this is to use `@StateObject`. That way only one instance is created during the view lifecycle.

Another thing is, to me it would make more sense to make it a property of `QueuedAdView` instead of `AdTypeSelectionView`, since it's closely related to the first one.